### PR TITLE
feat: appstudio: promote root image

### PR DESCRIPTION
**Pull request by psturc**
https://issues.redhat.com/browse/HACBS-788

### CODE REVIEW

File: ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml
Comment (psturc) [inline ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml:20]: should promote this image to openshift-ci registry after merge: `registry.ci.openshift.org/ci/redhat-appstudio:e2e-test-runner`

**Comment by psturc** [comment]
the rehearse job is failing on e2e test. however the issue is not related to this PR, because it doesn't change anything in the current e2e workflow.

@flacatus @jkopriva would you have a time to give this PR a look and add lgtm? Thanks

**Review by flacatus — APPROVED** [review]

**Comment by psturc** [comment]
I had to update this PR, because the prow config for build cluster for this job has changed. Could you please add another lgtm? Thanks
@flacatus @jkopriva

**Review by flacatus — APPROVED** [review]

**Pull request merged**
